.github/workflows/build.yml

```diff
@@ -39,6 +39,25 @@ jobs:
       - name: Build HAL for ${{ matrix.pac }}
         run: cargo check --package atsamx7x-hal --features ${{ matrix.pac }},unproven
 
+  build-latest:
+    needs: setup
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        pac: ${{ fromJson(needs.setup.outputs.pac_matrix) }}
+    continue-on-error: true # we let this fail and carry on, but GH UI is poor not showing amber like other CIs
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+      - name: Update toolchain
+        run: rustup update
+      - name: Specify toolchain
+        run: rustup override set stable
+      - name: Install Rust (thumbv7em)
+        run: rustup target add thumbv7em-none-eabihf
+      - name: Build HAL for ${{ matrix.pac }}
+        run: cargo check --package atsamx7x-hal --features ${{ matrix.pac }},unproven
```

Comment:
IIRC, not using a board feature (like `samv71q21b`) omits the build of some modules.

Reply:
@tmplt `${{ matrix.pac }}` is built from following output:
```
["same70j19b","same70j20b","same70j21b","same70n19b","same70n20b","same70n21b","same70q19b","same70q20b","same70q21b","sams70j19b","sams70j20b","sams70j21b","sams70n19b","sams70n20b","sams70n21b","sams70q19b","sams70q20b","sams70q21b","samv70j19b","samv70j20b","samv70n19b","samv70n20b","samv70q19b","samv70q20b","samv71j19b","samv71j20b","samv71j21b","samv71n19b","samv71n20b","samv71n21b","samv71q19b","samv71q20b","samv71q21b"]
```
and that's CI coming from you or your company.

Reply:
This looks good to me, although it feels like the CI is slowly growing more and more unruly and there should be a better way to do this.

Reply:
GitHub Action from Software Engineering perspective are utter nightmare. It's effectively a toy for JavaScript kids to develop their creativity, to serious tool for serious projects.